### ISSUE TITLE
Add the Missed Out TinyGo Technology

### ISSUE THREAD

**Issue by CoryGalyna** [ZORALab/Hestia#72]
# Description
```
Please provide a short description of what you have encountered below.
```
`TinyGo` was left out in the list of technologies. Please add it in.

# Link
```
Please provide the documentation link(s) below.
```
`/en`

# Expected Presentation
```
Please specify the expected presentation of the documentation.
```
TinyGo is listed.

# Current Presentation
```
Please specify the current presentation of the documentation.
```
TinyGo is missing.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
implemented in 677d2daf91a0af508ad25acd308508388f5cbdf9

**Comment by hollowaykeanho** [comment]
Published.
